tslib/src/functions/check-pin-expiry/check-pin-expiry.service.spec.ts

```diff
@@ -23,7 +23,6 @@ describe('CheckPinExpiryService', () => {
 
   test('modifies the database', async () => {
     await sut.process()
-    expect(sqlServiceMock.modify).toHaveBeenCalled()
-    expect(sqlServiceMock.modify.mock.calls[0][0]).toMatch(/^DELETE FROM \[mtc_admin\]\.\[checkPin\]/)
+    expect(sqlServiceMock.modify).toHaveBeenCalledWith(expect.any(String), [])
```

Comment:
I prefer the first test rather than the replacement.  Why the change?

Reply:
i changed that because it is verifying something that is not its concern (the underlying storage mechanism).  Ideally there should be a `checkPinExpiryDataService.deleteCheckPin` method which it could verify, but sorting that is out of scope.

However, the first/original test is more specific, and provides more value really, so i'm happy to change it back 👍